src/js/stores/history.js

```diff
@@ -638,6 +647,21 @@ define(function (require, exports, module) {
         },
 
         /**
+         * New and improved "unified" history event handler.  payload.history determines
+         * whether or not history state is created or amended.
+         *
+         * @param {object} payload
+         */
+        _handleUnifiedHistory: function (payload) {
+            if (payload.history && payload.history.newState) {
+                log.debug("UnifiedHistory: Creating new history state");
```

Comment:
I removed this debug log AND added a tag-like prefix.  Maybe someday we could have a way of enabling tag-level logging...